tests/define_test_aggregate_project.py

```diff
@@ -0,0 +1,77 @@
+from flow import FlowProject, aggregator, cmd, with_job
+
+
+class _TestAggregateProject(FlowProject):
```

Comment:
There's no need to make this internal. This is part of our testing suite, not the package.
```suggestion
class TestAggregateProject(FlowProject):
```

Reply:
This should not be changed, it matched the style of the other projects in our testing suite. However, it should be named like:
```suggestion
class _AggregateTestProject(FlowProject):
```

Reply:
Resolved in 7a8a806d52a8c6a01a8f32db90d087aa2e32b1fb.